docs/get-started/quick_start.md

```diff
@@ -1,8 +1,14 @@
 # Quick Start
 
-To compute on encrypted data, you first need to define the function you want to compute, then compile it into a Concrete `Circuit`, which you can use to perform homomorphic evaluation.
+This document covers how to compute on encrypted data homohorphically using the **Concrete** framework. We'll take you through a complete example step-by-step.
 
-Here is the full example that we will walk through:
+
+The basic workflow of computation is as follows:
+1. Define the function you want to compute
+2. Compile the function into a Concrete `Circuit`
+3. Use the `Circuit` to perform homomorphic evaluation
```

Comment:
Capitaliszation: **C**ircuit

Reply:
here, it must be "Circuit" with a C, it's code